3-project/README.md

```diff
@@ -2,20 +2,18 @@
 
 ## Objectives
 
-1. 
+1.
```

Comment:
Realising I never fill up this part, sorry for that. If you need me to explain the approach I took for these 3 chapter let me know, I'll be happy to share more.